backend/parser/examples/helloworld.cpp

```diff
@@ -13,46 +17,93 @@ class helloworld
 public:
 	helloworld();
 	void print();
-	~helloworld();
+	~helloworld(){
+	}
 	
 };
 
-int main()
-{	
+helloworld::helloworld(){
+
+}
+
+void lolasdasdasd(){
+}
+void lol(){
+    
+}
+
+int main(int argc, char** argv)
+{
+	int i;
+	int j;
+	bool ol;
+	bool lo;
+    helloworld hello = helloworld();
+	hello.print();
 	lolasdasdasd();
     std::cout << "Hello World!";
     return 0;
 }
 
+void (*signal(int, void (*fp)(int)))(int);
+
+auto hell() -> bool {
+    return true;
+}
+
+auto heaven() -> bool {
+    return false;
+}
 
-int lol(){
-	helloworld hw = new helloworld();
-	hw.print();
+int lolasdasdasd(int i, int j, bool istrue, float normalFloat){
+	helloworld* hw = new helloworld();
+
+	int z,w,l = 0;
+
+	std::vector<int> vInt;
+	std::vector<std::string> vString;
+	std::vector<bool> vBool;
+
+	std::unordered_map<std::string, std::string> map; 
+	std::unordered_map<int, std::vector<int>> map1; 
```

Comment:
Should probably be commented with an explanation saying that it is not handled by current grammar file.